src/main/groovy/com/palantir/gradle/shadowjar/ShadowJarPlugin.java

```diff
@@ -48,6 +48,8 @@ public class ShadowJarPlugin implements Plugin<Project> {
             groupOf("log4j"),
             groupOf("org.apache.logging.log4j"),
             groupOf("com.palantir.logsafe.logger"),
```

Comment:
huh, I think this line is incorrect, `com.palantir.logsafe.logger` isn't a maven coordinate (but _is_ a java package name), but `com.palantir.safe-logging` is: https://github.com/palantir/safe-logging/blob/f6276d51bf27f7f06da0cdb1dd8ac679a9c031b2/build.gradle#L37